lib/rouge/lexers/console.rb

```diff
@@ -103,7 +103,7 @@ def lang_lexer
       end
 
       def line_regex
-        /(\\.|[^\\])*?(\n|$)/m
+        /(\\(?:.|\Z)|[^\\])*?(\n|$)/m
```

Comment:
> it doesn't seem to interpret it as a line continuation. As far as I can see, it's unnecessary character matching.

I agree. Since line continuation is not supported, I think we can simplify the `line_regex` a bit.

```suggestion
        /(.*?)(\n|$)/
```

What do you think?

Reply:
Sure, that would certainly work. I wish I understood why it was trying to single out the backslash, but there are no tests to show what use case it was serving.

Reply:
I submitted an update. If you want it squashed, just let me know and I'll take care of it.